docs/maintainers.md

```diff
@@ -0,0 +1,68 @@
+# Maintainer's Handbook
+
+## Release Management
+
+The release procedure was recently changed from automatic to manual, due to too
+many problems with it.
+
+Here is a rough outline / checklist for the release:
+
+1. Checkout `master` branch, ensure you have all tags
+2. Prepare changelog
+3. Figure out the next version
+4. Update verion in the code
+5. Pull Request with the version bump.
+6. Create tag on the merge commit
+7. Upload / edit change log
+
+Here's how this works in detail:
+
+### Get release information
+
+The `semantic-release` tool can help you with the first few tasks of the above
+checklist:
+
+```bash
+# Ensure you're on the current master and have all release tags
+git checkout master
+git pull origin --tags
+
+# Prepare changelog
+semantic-release changelog --noop --unreleased -D version_source=tag
+
+# Figure out the next version
+semantic-release version --noop -D version_source=tag
+```
+
+You should verify that the version proposed by `semantic-release` is actually
+correct. At the time of writing (2020-01-22), it didn't detect some "BREAKING
+CHANGES" labels, for example. Update the changelog and version to what you think
+is correct.
+
+### Update version in source
+
+The version is also put in code. Update the file
+`caluma/caluma_metadata.py`.
+
+### Create a Pull request for the proposed version bump
+
+Put the changelog in the commit message or in the PR discussion somewhere, so
+it won't be forgotten once the release actually happens.
```

Comment:
Should we consider using `CHANGELOG.md`?

Reply:
I like the idea!

Reply:
Added CHANGELOG with historical entries